docs/src/visualization.md

```diff
@@ -85,6 +85,15 @@ that can be opened with any image viewer:
 
 !["solution_000040_scalar_resized"](https://github.com/trixi-framework/Trixi2Img.jl/raw/master/docs/src/assets/solution_000040_scalar_resized.png)
 
+To visualize 3D data generated by Trixi.jl, a 2D slice must be extracted.
+A slice can only lie in a plane orthogonal to one of the coordinate axes.
+The slice plane is defined by the axis to which it is orthogonal and an axis intercept.
+
+For example, to create a 2D slice in the yz-plane at the axis intercept `z = 0.5` from 3D data, execute
```

Comment:
Hah, you just failed my test of "please review my changes and only apply them if they make sense to you"! It must of course be the **xy** plane, not the **yz** plane...
```suggestion
For example, to create a 2D slice in the xy-plane at the axis intercept `z = 0.5` from 3D data, execute
```

But seriously, this was an error on my part, that's why it's really important that you review all changes proposed by me as well!

Reply:
Well, I reviewed your changes and missed that too. Perhaps Math is not my strong suit...

Reply:
Good thing you're not pursuing a career in mathematics or involving numbers in general :rofl: